### ISSUE TITLE
Name substitution in updated code is broken

### ISSUE THREAD

**Issue by NtTestAlert** [OlafvdSpek/xbt#62]
Example : tracker.cpp:290

` async_query("insert into " + db_name("files") + " (" + db_name("leechers") + ", " + db_name("seeders") + ", " + db_name("completed") + ", " + db_name("fid") + ") values "`

was changed to 
`async_query("insert into @files (@leechers, @seeders, @completed, @fid) values ?"`

but name substitution does find_first_of(" ,")

so it will look for name "fid)" find none and substitute with "fid)"

which will not work if the fid column is not named fid. 

similar with `@uid)` in a couple places

my workaround is to just put a space before ), but could also change name substitutor to handle )'s

for users using u232 for example this is probably causing issues ! users and torrents tables will not update (and possibly silently fail). possible side effect could be seeder/leecher/completed amounts not updating or !! downloaded amount not increasing in user table !!

**Comment by NtTestAlert** [comment]
same with 
```

				"  @leechers = values(@leechers),"
				"  @seeders = values(@seeders),"
				"  @completed = values(@completed),"
```

**Comment by OlafvdSpek** [comment]
Fixed in 210a39a8060c9dab657e6b88ed50baf954141514
